PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/StockModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  StockModel.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Tia Lendor on 9/5/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import Foundation
+
+enum StockJSONError: Error {
+    case decodingError(Error)
+}
+
+struct StockInfo: Codable {
+    
+    let date: String
+    let open: Double
+    let close: Double
+    let label: String
+    
+    static func getStockData(from data: Data) throws -> [StockInfo]{
+        do{
+            let stock = try JSONDecoder().decode([StockInfo].self, from: data)
+            return stock
+        } catch {
+            throw UserJSONError.decodingError(error)
+        }
+    }
+    
+
+    
```

Comment:
Make a function in this model that uses the open and close properties to determine whether there was in increase or a decrease. You can use this function your StocksDetailVC that changes the color and the image depending on what the output of the function is.